### ISSUE TITLE
Smaller data sample

### ISSUE THREAD

**Issue by ajw1970** [WeltyAutomation/BRTimeline#1]
For the sake of discussion and explanation, would it help to use the same data set for both the Data Log Visualization project and the BRTimeline? That way we could open them side by side and show the benefit of having the time gaps.

**Comment by thwoodley** [comment]
Yes, that would be ideal. Unfortunately the code I used to generate the time line data is for a previous iteration of log reader, it's one of the things I intended to do over the weekend but wasn't able to. I'll try to get it updated before the meeting.

Tyler Woodley
Application Engineer
Welty Automation, Inc.
www.WeltyAutomation.com<http://www.WeltyAutomation.com>
319-471-0781

________________________________

**Comment by ajw1970** [comment]
It might not be a priority if you have other things that you feel would be better to get done before 10am. I’ll let you decide on that.
